docs/logging/docs/020-architecture-logging.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Architecture
+type: Architecture
+---
+
+This document outlines the logging architecture of Kyma, highlighting information sources that Logspout extracts logs from containers and feeds them to OK Log.
```

Comment:
This document outlines the logging architecture of Kyma. It highlights information sources from which Logspout extracts logs to feeds to OK Log.

Reply:
done